docs/components/RecurringSelect/RecurringSelect.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
```

Comment:
Except for `Meta` - are these other imports used?

Reply:
oh, not anymore! keep forgetting my mdx doesn't delete those no mo :(

Reply:
Not meaning to start a review, @rodrigoeidelvein I think you claimed it... I just noticed this as I was looking at this... :)

Reply:
```suggestion
import { Meta } from "@storybook/addon-docs";
```